codalab/server/bundle_manager.py

```diff
@@ -692,6 +693,25 @@ def _get_staged_bundles_to_run(self):
                     bundle,
                     {'state': State.FAILED, 'metadata': {'failure_message': failure_message}},
                 )
+            elif bundle.metadata.request_queue:
+                # For those bundles that were requested to run on a private worker, if the private worker does not exist
+                # temporarily, we dispatch those bundles until the private worker connected to CodaLab server.
```

Comment:
Comment should make it clear that what we're doing here is simply updating the `staged_status` field, not actually carrying out any logic.